pages/styles.ts

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+
+export const BlockWrapper = styled.div`
+  display: flex;

```

Comment:
I suggest this block (4,5,6 lines) create few universal styles and using

Reply:
Done